src/bindings/Vscode.ml

```diff
@@ -502,8 +536,25 @@ module LanguageClient = struct
     -> t = "LanguageClient"
     [@@bs.new] [@@bs.module "vscode-languageclient"]
 
+  external start : t -> unit = "start" [@@bs.send]
+
+  external stop : t -> unit = "stop" [@@bs.send]
+
   external onReady : t -> unit Promise.t = "onReady" [@@bs.send]
 
+  external sendRequest : t -> string -> 'a -> 'b Promise.t = "sendRequest"
+    [@@bs.send]
+
+  external sendRequest_token :
+    t -> string -> 'a -> Workspace.cancellationToken -> 'b Promise.t
+    = "sendRequest"
+    [@@bs.send]
+
+  let sendRequest t ~meth ~data ?token () =
+    match token with
+    | None -> sendRequest t meth data
+    | Some token -> sendRequest_token t meth data token
+
```

Comment:
This section is necessary to avoid sending extra arguments to the sendRequest function which uses variable arguments.
See https://github.com/ocaml/ocaml-lsp/pull/233#discussion_r474338641